py-polars/polars/dataframe/frame.py

```diff
@@ -2473,14 +2474,15 @@ def write_csv(
     ) -> None:
         ...
 
+    @deprecate_renamed_parameter("quote", "quote_char", version="0.19.7")
```

Comment:
This will be included with the `0.19.8` release, so the version here should be corrected:

```suggestion
    @deprecate_renamed_parameter("quote", "quote_char", version="0.19.8")
```